tensorflow_datasets/core/dataset_info.py

```diff
@@ -265,6 +268,42 @@ def _dataset_info_path(self, dataset_info_dir):
   def _license_path(self, dataset_info_dir):
     return os.path.join(dataset_info_dir, LICENSE_FILENAME)
 
+  def visualize_dataset(self, disable_logging=True):
+    """Visualize dataset statistics with FACETS DIVE.
+
+       Example Usage:
+
+          titanic_builder = tfds.builder("titanic")
+          titanic_builder.download_and_prepare()
+          titanic_builder.info.visualize_dataset()
+
+       Return a <IPython.core.display.HTML object>
+
+       So it's just use on the jupyter notebook."""
+
+    filetype_suffix = self._builder._file_format_adapter.filetype_suffix
+
+    if filetype_suffix != "tfrecord":
+      raise ValueError(
+        "Cannot visualize dataset statistics because filetype {}".format(
+          filetype_suffix))
+
+    # Find the train .tfrecord files pattern of dataset.
+    filepattern = naming.filepattern_for_dataset_split(
+      self._builder.name, 'train', self._builder.data_dir, filetype_suffix)
+
+    statistics = LazyImporter.tfdv.\
+                              generate_statistics_from_tfrecord(filepattern)
```

Comment:
@Conchylicultor How can I get the generated statistics ?

Reply:
You can try to patch #782, then you should be able to use `info.splits['train'].statistics` directly.
 have issue with merging #782 as it doesn't works well with pytest-xdist and it significantly slow down the tests. Not sure yet what to do.

Reply:
Should I wait or merging this patch?

Reply:
You can test it on another branch if you want to test it, but I would probably wait for this PR.

Reply:
Okay I will wait PR.

Reply:
This is cool - can you also mention it in the main documentation ?

Is this PR ready to merge ?

Reply:
@cyfra There are unresolved  internal issues (see changelist 257065400).
I think it is best to wait for tfdv integration before merging this as it will simplify the statistics computations (the current code is not compatible with S3 I believe).
For tfdv, I'm waiting for the migration to S3 to complete and legacy split API has been removed.